beefy-gadget/src/lib.rs

```diff
@@ -194,6 +197,7 @@ struct BeefyWorker<Block: BlockT, Id, Signature, FinalityNotifications> {
 	signed_commitment_sender: BeefySignedCommitmentSender<Block, Signature>,
 	best_finalized_block: NumberFor<Block>,
 	best_block_voted_on: NumberFor<Block>,
+	validator_set_id: ValidatorSetId,
```

Comment:
I think this should be an `Option` and we should skip voting if it's not set. Otherwise I'm afraid that BEEFY validators will start signing bullshit commitments. Later on that should obviously be penalized, but currently I think it's safer to just wait for the beginning of the session. Not sure how this will work initially though (i.e. if we ever produce the first commitment).